CGPs/cgp-0005.md

```diff
@@ -5,5 +5,5 @@ date-created: 2020-06-11
 author: '@MarkusBerlin, @rcroessmann, @aslawson'
 status: EXPIRED
 governance-proposal-id: 5
-date-executed: NA
```

Comment:
empty fields are much more useful than fields with random, invalid data in them like `NA`, `TBD`, or other such variants